spec/api/client_spec.rb

```diff
@@ -2,6 +2,7 @@
 require "snitcher/api/client"
 require "base64"
 require "securerandom"
+require "uri"
```

Comment:
This require isn't necessary since we're not using `uri` any where in the test. I'm guessing this was added at one point during development but was refactored out.